channels/il.m3u

```diff
@@ -1,51 +1,31 @@
 #EXTM3U x-tvg-url="http://epg.streamstv.me/epg/guide-israel.xml.gz"
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="" group-title="",ch 12 קשת
+#EXTINF:-1 tvg-id="Channel 12 Keshet-IS" tvg-name="Channel 12 Keshet-IS" tvg-logo="https://img.mako.co.il/2019/10/03/newlogo12.jpg" group-title="עידן+",Keshet 12 קשת
```

Comment:
Wrong group-title. Here is a list of available groups: https://github.com/iptv-org/iptv#playlists-by-category

Reply:
Ok, thank you.
BTW what is the definition of: General, Auto, Other ?

Reply:
General - https://en.wikipedia.org/wiki/Generalist_channel
Auto - Channels about cars
Other - A category that is assigned automatically unless another category is specified